csharp/ql/lib/upgrades/dd813977f70fcbf737b0bbe9dc8297edff713168/upgrade.properties

```diff
@@ -1,4 +1,4 @@
-description: Added '@cil_function_pointer_type' and related relations ('cil_function_pointer_return_type',
-'cil_function_pointer_calling_conventions'). Introduced 'cil_type_annotation' and '@cil_has_type_annotation'
+description: Added '@cil_function_pointer_type' and related relations ('cil_function_pointer_return_type', \
```

Comment:
Huh, I had no idea these had to be single lines.

Reply:
The [java.util.Properties line format](https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html#load-java.io.Reader-) is extremely flexible.  Each line is a key-value pair, with the key and the value separated by `=`, `:`, `:=`, `=:`, `==`, `::`, or just blank space.  So two original lines would be parsed as:

- Key `description` to value `Added '@cil_function_pointer_type' and related relations ('cil_function_pointer_return_type',`
- Key `'cil_function_pointer_calling_conventions').` to value `Introduced 'cil_type_annotation' and '@cil_has_type_annotation'`